src/timepicker/timepicker.spec.ts

```diff
@@ -360,10 +360,40 @@ describe('ngb-timepicker', () => {
        })));
 
   });
+
+  describe('disable', () => {
+
+    it('should not change the value, when it is disabled', async(inject([TestComponentBuilder], (tcb) => {
```

Comment:
We should also check that a user is not allowed to change the value of any of the input fields directly (by entering a new value).